### PR TITLE
Implementing Catalog Content View skeleton

### PR DESCRIPTION
**Pull request by AnnaBanani**
CatalogContentView class created
CatalogCell class created
VerticalGradientView created

### CODE REVIEW

**Review by Ssimboss — CHANGES_REQUESTED** [review]

File: TheMET/TheMET/Network/LoadingImageView.swift [hunk 1]
```diff
@@ -45,7 +45,7 @@ class LoadingImageView: UIView {
         self.addSubview(self.imageView)
         self.addSubview(self.activityIndicator)
         self.addSubview(self.failedImageView)
-        self.imageView.contentMode = .scaleAspectFit
+        self.imageView.contentMode = .scaleToFill
```
[the remark this answers is not in the document]
Comment (AnnaBanani) [reply TheMET/TheMET/Network/LoadingImageView.swift:48]: fixed

File: TheMET/TheMET/Extentions/UIView+applyGradient.swift [hunk 2]
```diff
@@ -0,0 +1,9 @@
+//
+//  UIView+applyGradient.swift
+//  TheMET
+//
+//  Created by Анна Ситникова on 12/05/2023.
+//
+
+import Foundation
```
[the remark this answers is not in the document]
Comment (AnnaBanani) [reply TheMET/TheMET/Extentions/UIView+applyGradient.swift:8]: fixed

File: TheMET/TheMET/Network/LoadingImageView.swift [hunk 3]
```diff
@@ -45,7 +45,6 @@ class LoadingImageView: UIView {
         self.addSubview(self.imageView)
         self.addSubview(self.activityIndicator)
         self.addSubview(self.failedImageView)
-        self.imageView.contentMode = .scaleAspectFit
```
Comment (Ssimboss) [inline TheMET/TheMET/Network/LoadingImageView.swift:48]: Why `self.imageView.contentMode` is not set anymore?

**Review by Ssimboss — CHANGES_REQUESTED** [review]

File: TheMET/TheMET/Network/LoadingImageView.swift (hunk 3 above)
Comment (AnnaBanani) [reply TheMET/TheMET/Network/LoadingImageView.swift:48]: fixed

**Review by Ssimboss — APPROVED** [review]

**Pull request merged**